go.mod

```diff
@@ -23,8 +23,12 @@ require (
 	go.opencensus.io v0.24.0
 	golang.org/x/net v0.10.0
 	golang.org/x/oauth2 v0.8.0
+	golang.org/x/sync v0.2.0
+	golang.org/x/sys v0.8.0
+	golang.org/x/text v0.9.0
 	golang.org/x/time v0.3.0
 	google.golang.org/api v0.126.0
+	gopkg.in/yaml.v2 v2.4.0
```

Comment:
Could you please help me understand why we are moving this from indirect to direct ?

Reply:
We are using directly [here](https://github.com/GoogleCloudPlatform/gcsfuse/blob/66b8100310ff701a7f4fba5f801615c3480228dc/tools/integration_tests/util/setup/yaml-config.go#L9).